Source/DafnyCore/Compilers/CSharp/Compiler-Csharp.cs

```diff
@@ -79,7 +79,11 @@ public class CsharpCompiler : SinglePassCompiler {
         CsharpSynthesizer.EmitImports(wr);
       }
       EmitDafnySourceAttribute(program, wr);
-      ReadRuntimeSystem(program, "DafnyRuntime.cs", wr);
+
+      if (DafnyOptions.O.IncludeRuntime) {
```

Comment:
FYI this doesn't change anything, because `ReadRuntimeSystem` checks for the option and returns early itself.

Reply:
Not anymore :D